src/components/panel/panel.js

```diff
@@ -126,6 +126,9 @@ angular
  *     outside the panel to close it. Defaults to false.
  *   - `escapeToClose` - `{boolean=}`: Whether the user can press escape to
  *     close the panel. Defaults to false.
+ *   - `postClose` - `{function=}`: Function that is called after close
+ *     is done. The first parameter passed into this function is the closeEvent
```

Comment:
Possible re-word to "after close has successfully finished". "done" is too vague for this, since it will not be called if the panel fails the close.